src/VisualStudio/Core/Test/CodeModel/CSharp/EventCollectorTests.vb

```diff
@@ -811,7 +811,86 @@ class C
 </Code>
 
             Test(code, changedCode,
-                 ArgChange("System.CLSCompliant"))
+                 ArgChange("System.CLSCompliant", "foo"),
+                 ArgChange("System.CLSCompliant", "bar"))
+        End Sub
+
+        <WorkItem(844611)>
+        <ConditionalFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModelEvents)>
+        Public Sub TestField_AddOneMoreAttribute()
+            Dim code =
+<Code>
+using System;
+
+class Program
+{
+    [Foo]
+    public int bar;
+}
+
+public class FooAttribute : Attribute
```

Comment:
Is it important that this be an attribute from source?  If not, maybe remove in favor of a built-in one to reduce noise in these tests?